### PR TITLE
Add block-v3 SSZ tests

### PR DESCRIPTION
**Pull request by eserilev**
## Proposed Changes

Add ssz tests for the block v3 endpoint. These tests were missed in the original block v3 PR. Also, these changes are required in this PR https://github.com/blockprint-collective/blockdreamer/pull/5 to test the block v3 ssz flow in blockdreamer

### CODE REVIEW

File: common/eth2/src/lib.rs [hunk 1]
```diff
@@ -1729,6 +1738,42 @@ impl BeaconNodeHttpClient {
...
 
+        Ok(path)
+    }
+
+    /// `GET v3/validator/blocks/{slot}`
+    pub async fn get_validator_blocks_v3<T: EthSpec>(
+        &self,
+        slot: Slot,
+        randao_reveal: &SignatureBytes,
+        graffiti: Option<&Graffiti>,
```
Comment (realbigsean) [inline common/eth2/src/lib.rs:1749]: `skip_randao_verification` should be optionally passed in here, right?

File: common/eth2/src/lib.rs [hunk 2]
```diff
@@ -1750,6 +1795,58 @@ impl BeaconNodeHttpClient {
         }
     }
 
+    /// `GET v3/validator/blocks/{slot}` in ssz format
+    pub async fn get_validator_blocks_v3_ssz<T: EthSpec>(
+        &self,
+        slot: Slot,
+        randao_reveal: &SignatureBytes,
+        graffiti: Option<&Graffiti>,
```
Comment (realbigsean) [inline common/eth2/src/lib.rs:1803]: Here too

**Review by realbigsean — APPROVED** [review]
Nice!

File: common/eth2/src/lib.rs (hunk 1 above)
Comment (michaelsproul) [reply common/eth2/src/lib.rs:1749]: There's the `_modular` variant with the randao verification param. Seems consistent with the other functions as-is.

Comment (realbigsean) [reply common/eth2/src/lib.rs:1749]: gotcha!

**Pull request merged**